src/pages/transactions/_components/DfTx/DfTxSetFutureSwap.tsx

```diff
@@ -20,6 +25,7 @@ export function DfTxSetFutureSwap(props: DfTxSetFutureSwapProps): JSX.Element {
           fromTokenId={from?.token}
           fromAmount={from?.amount}
           toTokenId={to}
+          destToken={destToken}
```

Comment:
instead of re-introducing a new props, can we pass the token ID of `DUSD` to `toTokenId` when the condition is met?

Reply:
you can refactor the condition as a local boolean state, the state name can be used to help to explain to other code reader why we are doing this

Reply:
we cannot pass the token ID of DUSD to toTokenId because the token ID is not a fixed token ID eg 0 is always for DFI but DUSD is not always 11 it can be 15 or 12

Reply:
updated codes based on comments added in switch case to check DUSD values